lib/index.js

```diff
@@ -54,17 +55,17 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
         account, email, hash, clientId, serviceId,
         authenticationMethod, requiredAuthenticationMethods
       } = req.body;
-
       if(type === 'nonce') {
-        // permitted without authentication
+        // permitted without authentication. Do not require an account since
+        // anyone can request for a nonce.
         const {actor = null} = (req.user || {});
         await brAuthnToken.set({
           actor, account, email, type, clientId,
           authenticationMethod, requiredAuthenticationMethods
         });
         res.status(204).end();
       } else {
-        // all other types require authentication to create
+        // all other types require authentication to create(Require an account)
```

Comment:
the additional parenthetical here make things less clear to me rather than more clear.  Please reword to make the meaning more clear.  Although I'm not sure specifically calling out that an "account" is required, something like this might be better:
```suggestion
        // all other types require an authenticated account to create
```

Reply:
Updated.